src/inmanta/server/services/compilerservice.py

```diff
@@ -336,20 +350,27 @@ async def install_modules() -> None:
             LOGGER.exception("An error occured while recompiling")
 
         finally:
+
+            async def warn(message: str) -> None:
```

Comment:
Why is this async?

Reply:
Because `self._warning` is async, I forgot the `await`. I'm surprised this didn't trigger any test failures.

Reply:
(fixed)